tests/testthat/test-data_tabulate.R

```diff
@@ -508,3 +508,41 @@ test_that("data_tabulate, as.data.frame, cross tables with total N", {
   expect_identical(nrow(out$table[[1]]), 5L)
   expect_identical(out$table[[1]]$cyl, c("4", "6", "8", NA, "Total"))
 })
+
+
+# table methods -----------------------------
+
+test_that("data_tabulate, table methods", {
```

Comment:
It would be much easier to review snapshot tests (also to check potential changes in the future)

Reply:
I can also add some snapshot tests, but would still keep the existing ones as well.